packages/coinjoin/tests/backend/CoinjoinBackendClient.test.ts

```diff
@@ -9,11 +10,40 @@ const MALFORMED_HASH = 'deadbeef';
 const VALID_TX = 'd20a1e1f3b98f82dc7aa0fa0538b75be357ab53f2d1c6f68c0be4e7537122f5d';
 const INVALID_TX = 'deadbeef';
 
-describe.skip('CoinjoinBackendClient', () => {
+const BLOCKBOOKS = ['bb_A', 'bb_B', 'bb_C', 'bb_D', 'bb_E', 'bb_F', 'bb_G'];
+
+describe('CoinjoinBackendClient', () => {
     let client: CoinjoinBackendClient;
 
     beforeAll(() => {
-        client = new CoinjoinBackendClient(COINJOIN_BACKEND_SETTINGS);
+        client = new CoinjoinBackendClient({
+            ...COINJOIN_BACKEND_SETTINGS,
+            blockbookUrls: BLOCKBOOKS,
+        });
+    });
+
+    it.only('blockbook backends rotation', async () => {
```

Comment:
only?

Reply:
Yes, that's the thing. The whole test set was skipped before as it had been testing against real backend servers rather than some mocks. What to do?
- remove the remaining tests
- mock server responses (but then the tests won't be testing much)
- something else?

Reply:
i would use ./tests/fixtures/server to mock server responses.
why do you think it won't be testing much? by doing this you can test everything you want/need including server errors/unexpected states

Reply:
but i understand you dont want to do it right now :) resolving